website/docs/components/icon-tile/partials/guidelines/guidelines.md

```diff
@@ -6,56 +6,47 @@ To represent an object or as part of a page title.
 
 ### When not to use
 
-Within a paragraph where an object is referred to.
+- Within a paragraph where an object is referred to.
```

Comment:
I'm not really sure what you mean by this, can you clarify a bit?

Reply:
I can remove this snippet of text. I think what I was trying to say is, "Don't use outside of headings and standalone visuals." in a vague way 😅